### PR TITLE
Column addition and deletion in various tables of the plugin.

### PR DESCRIPTION
**Pull request by sidr0cker**
# Example query results
<details>
  <summary>Results</summary>
  
```
Add example SQL query results here (please include the input queries as well)
```
</details>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,38 @@
+## v1.0.0 [2022-08-12]
+
+_What's new?_
+
+- New tables added
+  - [steampipecloud_workspace_snapshot](https://hub.steampipe.io/plugins/turbot/steampipecloud/tables/steampipecloud_workspace_snapshot) ([#22](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/22))
+  - [steampipecloud_workspace_db_log](https://hub.steampipe.io/plugins/turbot/steampipecloud/tables/steampipecloud_workspace_db_log) ([#23](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/23))
+
+_Enhancements_
```
Comment (LalitLab) [inline CHANGELOG.md:9]: I think we need to call out the removed columns separately in the `Breaking changes` section in CHANGELOG.md

Comment (LalitLab) [reply CHANGELOG.md:9]: For reference https://github.com/turbot/steampipe-plugin-azure/blob/main/CHANGELOG.md#v020-2021-02-18

**Review by LalitLab — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,41 @@
...
+- Added `identity_handle`, `workspace_handle`, `created_by_id`, `created_by`, `updated_by_id`, `updated_by`, `version_id` columns to `steampipecloud_workspace_mod` table. ([#24](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/24))
+- Added `version_id` column to `steampipecloud_workspace_mod_variable` table. ([#24](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/24))
+
+_Breaking changes_
+
+- Removed column `email` from `steampipecloud_organization_member` table. ([#24](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/24))
+- Removed column `email` from `steampipecloud_organization_workspace_member` table. ([#24](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/24))
+- Removed column `email` from `steampipecloud_user` table. ([#24](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/24))
+- Modified column `workspace_state` renamed to `state` in `steampipecloud_workspace` table. ([#24](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/24))
```
Comment (cbruno10) [inline CHANGELOG.md]:
```suggestion
- Renamed column `workspace_state` to `state` in `steampipecloud_workspace` table. ([#24](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/24))
```

I think we can shorten this note, and others like it

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,41 @@
...
+- Modified column `mod_constraint` renamed to `constraint` in `steampipecloud_workspace_mod` table. ([#24](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/24))
+- Modified column `created_by_handle` renamed to `created_by_id` to store identifier of the person who created the setting in `steampipecloud_workspace_mod_variable` table. ([#24](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/24))
```
Comment (cbruno10) [inline CHANGELOG.md]:
```suggestion
- Renamed column `created_by_handle` to `created_by_id` and updated the column to store the identifier of the person who created the variable in `steampipecloud_workspace_mod_variable` table. ([#24](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/24))
```

Can you please update other items like this as well?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,41 @@
...
+  - [steampipecloud_workspace_snapshot](https://hub.steampipe.io/plugins/turbot/steampipecloud/tables/steampipecloud_workspace_snapshot) ([#22](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/22))
+  - [steampipecloud_workspace_db_log](https://hub.steampipe.io/plugins/turbot/steampipecloud/tables/steampipecloud_workspace_db_log) ([#23](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/23))
+
+_Enhancements_
+
+- Added `process_id` column to `steampipecloud_audit_log` table. ([#24](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/24))
+- Added `created_by_id`, `created_by`, `updated_by_id`, `updated_by` columns to `steampipecloud_connection` table. ([#24](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/24))
+- Added `created_by_id`, `updated_by_id` columns to `steampipecloud_organization` table. ([#24](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/24))
+- Modified columns `created_by`, `updated_by` to store additional information about the user who created or updated the organization in `steampipecloud_organization` table. ([#24](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/24))
```
Comment (cbruno10) [inline CHANGELOG.md]:
```suggestion
- Updated columns `created_by`, `updated_by` to store additional information about the user who created or updated the organization in `steampipecloud_organization` table. ([#24](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/24))
```

We consistently use `updated` more than `modified` in CHANGELOG.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,41 @@
+## v0.4.0 [2022-08-19]
+
+_What's new?_
+
+- New tables added
+  - [steampipecloud_workspace_snapshot](https://hub.steampipe.io/plugins/turbot/steampipecloud/tables/steampipecloud_workspace_snapshot) ([#22](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/22))
```
Comment (cbruno10) [inline CHANGELOG.md]: Can you please order these alphabetically?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,41 @@
...
+  - [steampipecloud_workspace_db_log](https://hub.steampipe.io/plugins/turbot/steampipecloud/tables/steampipecloud_workspace_db_log) ([#23](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/23))
+
+_Enhancements_
+
+- Added `process_id` column to `steampipecloud_audit_log` table. ([#24](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/24))
+- Added `created_by_id`, `created_by`, `updated_by_id`, `updated_by` columns to `steampipecloud_connection` table. ([#24](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/24))
+- Added `created_by_id`, `updated_by_id` columns to `steampipecloud_organization` table. ([#24](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/24))
+- Modified columns `created_by`, `updated_by` to store additional information about the user who created or updated the organization in `steampipecloud_organization` table. ([#24](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/24))
+- Added `user`, `scope`, `created_by_id`, `updated_by_id` columns to `steampipecloud_organization_member` table. ([#24](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/24))
```
Comment (cbruno10) [inline CHANGELOG.md]: When there are multiple columns, can you please order alphabetically?

File: steampipecloud/table_steampipecloud_workspace_mod_variable.go [hunk 7]
```diff
@@ -89,32 +89,38 @@ func tableSteampipeCloudWorkspaceModVariable(_ context.Context) *plugin.Table {
...
 			{
-				Name:        "updated_at",
-				Description: "Time when the mod was updated.",
-				Type:        proto.ColumnType_TIMESTAMP,
-			},
-			{
-				Name:        "created_by_handle",
-				Description: "Handle of the user who created the setting if any.",
+				Name:        "created_by_id",
+				Description: "Unique Identifier of the user who created the setting.",
```
Comment (cbruno10) [inline steampipecloud/table_steampipecloud_workspace_mod_variable.go]:
```suggestion
				Description: "Unique identifier of the user who created the setting.",
```

File: steampipecloud/table_steampipecloud_workspace_mod_variable.go [hunk 8]
```diff
@@ -89,32 +89,38 @@ func tableSteampipeCloudWorkspaceModVariable(_ context.Context) *plugin.Table {
...
 				Type:        proto.ColumnType_JSON,
 			},
+			{
+				Name:        "updated_at",
+				Description: "Time when the mod was updated.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "updated_by_id",
+				Description: "Unique Identifier of the user who last updated the setting.",
```
Comment (cbruno10) [inline steampipecloud/table_steampipecloud_workspace_mod_variable.go]:
```suggestion
				Description: "Unique identifier of the user who last updated the setting.",
```

File: steampipecloud/table_steampipecloud_workspace_mod_variable.go [hunk 9]
```diff
@@ -89,32 +89,38 @@ func tableSteampipeCloudWorkspaceModVariable(_ context.Context) *plugin.Table {
...
+				Transform:   transform.FromCamel(),
 			},
 			{
 				Name:        "created_by",
 				Description: "Information about the user who created the Setting.",
 				Type:        proto.ColumnType_JSON,
 			},
+			{
+				Name:        "updated_at",
+				Description: "Time when the mod was updated.",
```
Comment (cbruno10) [inline steampipecloud/table_steampipecloud_workspace_mod_variable.go]: Is this column for when the mod or mod variable was updated?

File: steampipecloud/table_steampipecloud_workspace.go [hunk 10]
```diff
@@ -46,73 +46,97 @@ func tableSteampipeCloudWorkspace(_ context.Context) *plugin.Table {
...
+			},
+			{
+				Name:        "created_by_id",
+				Description: "The unique identifier of the user who created the workspace.",
+				Type:        proto.ColumnType_STRING,
 				Transform:   transform.FromCamel(),
 			},
+			{
+				Name:        "created_by",
+				Description: "Information about the user who created the workspace..",
```
Comment (cbruno10) [inline steampipecloud/table_steampipecloud_workspace.go]:
```suggestion
				Description: "Information about the user who created the workspace.",
```

**Review by cbruno10 — CHANGES_REQUESTED** [review]
@siddharthaturbot Please see review comments, thanks!

File: CHANGELOG.md (hunk 3 above)
Comment (sidr0cker) [reply CHANGELOG.md]: We actually store the identifier of the user who created a setting on the variable. The variables are created by steampipe cloud.

Comment (cbruno10) [reply CHANGELOG.md]: Oh ok, I see, thanks!

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**